hugo/assets/scss/components/note.scss

```diff
@@ -1,31 +1,62 @@
 @import '../config/colors';
 @import '../mixins/marginless-child';
+@import '../mixins/screen';
+@import '../mixins/svg';
 
 .note {
     $self: &;
 
-    --note-bg-color: #{ $c-blue--lighter };
```

Comment:
for themed components it's best to set the variables at the top for the default colors so it's clear what variables are themed. So these should not be removed here. For themed components we set the color variables at the top.